code/src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -57,6 +58,7 @@
     private SwerveDriveKinematics m_kinematics;
     private AHRS m_gyro;
     private PIDController m_pidController;
+    private SwerveDriveOdometry m_currentOdometry;
```

Comment:
any reason why it's called current odomentry as opposed ot `m_odomentry`?

Reply:
Not really, I used current odometry to show that, that was the current pose of the robot at the end of the move command. Should I change it?